test/unit/mocks/browser.js

```diff
@@ -190,4 +194,22 @@ window.browser = {
     registerScalars(category, scalars) {},
     scalarSet(name, value) {},
   },
+
+  experiments: {
+    logins: {
+      async getAll() { return []; },
+      async add(login) {
+        browser.experiments.logins.onAdded.getListener()({ login });
```

Comment:
I was hoping I could be less verbose here by just using `this`, but that apparently ends up being the wrong reference

Reply:
🤔